samples/appendable-template/ml_service/pipelines/diabetes_regression_build_train_pipeline_appendable.py

```diff
@@ -0,0 +1,222 @@
+        aml_workspace,
+        e.aml_env_name,
+        conda_dependencies_file=e.aml_env_train_conda_dep_file,
+        create_new=e.rebuild_env,
+    )  #
+    run_config = RunConfiguration()
+    run_config.environment = environment
+
+    if e.datastore_name:
+        datastore_name = e.datastore_name
+    else:
+        datastore_name = aml_workspace.get_default_datastore().name
+    run_config.environment.environment_variables[
+        "DATASTORE_NAME"
+    ] = datastore_name  # NOQA: E501
+
+    model_name_param = PipelineParameter(name="model_name", default_value=e.model_name)  # NOQA: E501
+    dataset_version_param = PipelineParameter(
+        name="dataset_version", default_value=e.dataset_version
+    )
+    data_file_path_param = PipelineParameter(
+        name="data_file_path", default_value="none"
+    )
+    caller_run_id_param = PipelineParameter(name="caller_run_id", default_value="none")  # NOQA: E501
+
+    pipeline_parameters = {
+        "model_name": model_name_param,
+        "dataset_version": dataset_version_param,
+        "data_file_path": data_file_path_param,
+        "caller_run_id": caller_run_id_param,
+    }
+
+    # Get dataset name
+    dataset_name = e.dataset_name
+
+    # Check to see if dataset exists
+    if dataset_name not in aml_workspace.datasets:
+        # This call creates an example CSV from sklearn sample data. If you
+        # have already bootstrapped your project, you can comment this line
+        # out and use your own CSV.
+        create_sample_data_csv()
+
+        # Use a CSV to read in the data set.
+        file_name = "diabetes.csv"
+
+        if not os.path.exists(file_name):
+            raise Exception(
+                'Could not find CSV dataset at "%s". If you have bootstrapped your project, you will need to provide a CSV.'  # NOQA: E501
+                % file_name
+            )  # NOQA: E501
+
+        # Upload file to default datastore in workspace
+        datatstore = Datastore.get(aml_workspace, datastore_name)
+        target_path = "training-data/"
+        datatstore.upload_files(
+            files=[file_name],
+            target_path=target_path,
+            overwrite=True,
+            show_progress=False,
+        )
+
+        # Register dataset
+        path_on_datastore = os.path.join(target_path, file_name)
+        dataset = Dataset.Tabular.from_delimited_files(
+            path=(datatstore, path_on_datastore)
+        )
+        dataset = dataset.register(
+            workspace=aml_workspace,
+            name=dataset_name,
+            description="diabetes training data",
+            tags={"format": "CSV"},
+            create_new_version=True,
+        )
+
+    # Create a PipelineData to pass data between steps
+    pipeline_data = PipelineData(
+        "pipeline_data", datastore=aml_workspace.get_default_datastore()
+    )
+
+    # List of pipeline steps
+    steps = list()
+
+    # train_step = PythonScriptStep(
```

Comment:
How about removing the commented code part coming from the original example? I feel that it just creates confusion. The way it works is already really good described in the README.

Reply:
The commented code removed.